project-roles/trusted-committer.md

```diff
@@ -0,0 +1,162 @@
+# Title
+
+Trusted Comitter
```

Comment:
In open source, people typically use "committer" to refer to the person that sits on the repository, merges code contributions, and mentors developers by giving them feedback on their contributions. This pattern (and also many of us at the commons) use the term "_trusted_ committer" instead of "committer".

This implies there is a difference between committer and trusted committer. 

I'd suggest to add to this pattern:  What is the "trusted" in "trusted committer"? What is the difference to a committer - which is the established term from open source?

Reply:
Or alternatively: Why not dropping the term _trusted_ in _trusted committer_?

Reply:
We've heard this term used quite a bit in the commons - just bringing the term forward here.  Is there someone in the commons that has background on where the term came from?

Reply:
Maybe Danese or @Silona can help. They were the first I heard adding _trusted_ :)

Reply:
I have never heard the term outside of the commons. Here in my little part of the world, the role which we termed _Trusted Committer_ is _Maintainer_. The maintainer role is also mentioned in the Patlet and to me it sounds like the _Trusted Committer_ is somewhere between a maintainer and the committer in rank. That (to me) sounds like one hierarchy level to much.

Reply:
Thanks for all this feedback! Here's my take on TC vs Maintainer:

Being the maintainer of a project carries the implication of total responsibility (roadmap definition, feature prioritization, delivery). Other teams have different priorities, and it's likely that becoming the maintainer of a library isn't an option for many external contributors. We feel the Trusted Committer role fills a niche that sits between occasional contributor and maintainer, someone who knows how the project works and is able to mediate things like pull requests or issue triage without being on the hook for driving the roadmap, larger feature work, etc.

If I were to create a rough hierarchy of how are projects are structured from an engineering viewpoint it'd look something like this:

1. Maintainers
    - The team that is innersourcing their library, service, or application
1. Trusted Committers
    - Consistent contributors that can triage issues, commit to master, release, etc.
1. Contributors
    - Occasional features, fixes, etc. mostly users that are making things work for them
1. Users
    - Most folks that are using a team's library/service, maybe filing issues here and there

Reply:
I think this makes sense.  They have more of a role than contributor but less than maintainer.  It sounds like giving a short nod to these distinctions would be useful.

Reply:
As for where the term comes from, the first time I read the term "Trusted Committer" was here: http://paypal.github.io/InnerSourceCommons/assets/files/InnerSourceChecklist.pdf

@dogonthehorizon In your hierarchy - is it possible for a Trusted Committer to become a Maintainer?

Reply:
@MaineC definitely, although we haven't found a situation where that makes sense yet. There's likely another patlet here around responsible transfer of ownership for innersourced projects (i.e. what happens when the originating team can no longer maintain the project? Who owns it? How do things get transferred?)